icelake/src/io/data_file_writer.rs

```diff
@@ -48,6 +51,7 @@ impl DataFileWriter {
         arrow_schema: SchemaRef,
         rows_divisor: usize,
         target_file_size_in_bytes: u64,
```

Comment:
Does these two parameter should also included in TableConfig

Reply:
Yeah, I think so.